tox.ini

```diff
@@ -364,11 +364,14 @@ deps =
     -r dev-requirements.txt
     -r benchmarks/scripts/requirements.txt
     -r benchmarks/micro/requirements-compression-algorithms.txt
-    pytest-benchmark[histogram]
+    # need < 9.0.0 for python 2.7
+    py-cpuinfo==8.0.0
```

Comment:
hmm, I've tried to install various earlier versions of `py-cpuinfo` but got same error.

Reply:
Yeah, multiple dependencies were causing problems, not just py-cpuinfo (also pygal, etc.).